app/scripts/modules/titus/src/pipeline/stages/runJob/TitusRunJobStageConfig.tsx

```diff
@@ -218,7 +236,69 @@ export class TitusRunJobStageConfig extends React.Component<IStageConfigProps, I
           onChange={this.dockerChanged}
           deferInitialization={stage.deferredInitialization}
         />
-
+        <h4>
```

Comment:
It would be nice to extract all the EFS options to a separate component

Reply:
Agree and i am thinking to get this out first and re-visit the `EFS` component as part of migration where user can specify multiple mounts. Let me know your thoughts ?

Reply:
sounds good!